templates/named.conf.erb

```diff
@@ -24,5 +24,12 @@ acl "<%= acl_name %>"  {
 };
 <%- end -%>
 
+<%- if scope.lookupvar('::dns::additional_directives').any? -%>
+// additional directives
+<% scope.lookupvar('::dns::additional_directives').each do |directive| -%>
+<%= directive %>
+<%- end -%>
+<%- end -%>
```

Comment:
The spec test in dns_init_spec.rb line 36 still goes through, so there will be no change for users after the module update.    Is that what you mean?